### PR TITLE
[staging] Replace celery machine

### PR DESCRIPTION
**Pull request by skodde**
https://dimagi-dev.atlassian.net/browse/SAAS-13221

##### ENVIRONMENTS AFFECTED
Staging

### CODE REVIEW

File: environments/staging/aws-resources.yml [hunk 1]
```diff
@@ -40,10 +42,6 @@ pgbouncer0-staging.instance_id: i-0315ba6f2f8ba45f5
 pgformplayer0-staging: pgformplayer0-staging.czkracjslrn2.us-east-1.rds.amazonaws.com
 pgformplayer_nlb-staging: pgformplayer-nlb-staging-98b68467f6f5717b.elb.us-east-1.amazonaws.com
 pgmain_nlb-staging: pgmain-nlb-staging-fe77ab2d143eedbb.elb.us-east-1.amazonaws.com
-pgproxy2-staging: 10.201.40.16
-pgproxy2-staging.instance_id: i-0638ff6218ed9f753
-pgproxy3-staging: 10.201.40.79
-pgproxy3-staging.instance_id: i-0ceb5822e286daf7e
```
Comment (gherceg) [inline environments/staging/aws-resources.yml:46]: assuming this is just unrelated cleanup?

File: environments/staging/app-processes.yml [hunk 2]
```diff
@@ -1,31 +1,39 @@
...
+      prefetch_multiplier: 1
+    malt_generation_queue:
+      concurrency: 1
     reminder_queue:
       pooling: gevent
       concurrency: 4
     saved_exports_queue:
       concurrency: 3
-      optimize: True
+      prefetch_multiplier: 1
```
Comment (gherceg) [inline environments/staging/app-processes.yml:36]: mildly curious what this entails? Though I trust you've added it for good reason.

**Review by gherceg — APPROVED** [review]

File: environments/staging/aws-resources.yml (hunk 1 above)
Comment (skodde) [reply environments/staging/aws-resources.yml:46]: Yes, it came with `aws-fill-inventory`, probably a leftover from another PR/maintenance.

File: environments/staging/app-processes.yml (hunk 2 above)
Comment (skodde) [reply environments/staging/app-processes.yml:36]: Regarding `optimize`, it's no longer in use, I'll make a PR to clean up the dead code.
